### ISSUE TITLE
Update Java Build Tools to work with new server

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/tasks#672]
- [ ] Deploying new Java translations through PBG and building them on the server
- [ ] .jnlp files should link to HTTPS instead of HTTP 
- [ ] PBG should point to phet-server instead of figaro

**Comment by jbphet** [comment]
Marked for developer meeting so we can determine who will work on this and when.

**Comment by pixelzoom** [comment]
8/18/16 dev meeting:
• Java translations can't be published until this work is done
• @mattpen has other things that need to be done first related to server cut-over

**Comment by mattpen** [comment]
While also updating the tools we should also make them compatbile with JDK 8 which is currently running on phet-server, or making the JDK 7 binaries available.

**Comment by phet-steele** [comment]
@mattpen @jbphet @ariel-phet this is the weekly reminder I was told to give!

**Comment by mattpen** [comment]
Unassigning until work resumes.

**Comment by mattpen** [comment]
@jonathanolson and I worked on updating the PBG to deploy sims and translations to phet-server today.  We tested deploying a translation for Ideal Gas, and it appears to have worked.  

@phet-steele - Can you please check if the deploy was successful?

**Comment by phet-steele** [comment]
@jonathanolson @mattpen the translation for Ideal Gas appeared to go ideally (publishing also felt faster 😄). It may be worth it to also try a Flash/Flex sim translation.

**Comment by zepumph** [comment]
In phetsims/website#142 there was a possibility of adding feedback to the process of publishing java sims' translations. Can we do that while working on this? I'd love to help if you need anything.

**Comment by mattpen** [comment]
The build tools were updated for the new server in this changeset: https://phet.unfuddle.com/a#/repositories/23262/commit?commit=74659.

We've tested redeploying the jars for Java and Flash sims as part of the resigning process, so that task is complete.

> adding feedback to the process of publishing java sims' translations

@zepumph - Since the original work is completed this should be opened in a new issue.
